jwt.cabal

```diff
@@ -33,7 +33,7 @@ library
   exposed-modules:     Web.JWT
   other-modules:       Data.Text.Extended, Data.ByteString.Extended
   build-depends:       base >= 4.8 && < 5
-                     , cryptonite               >= 0.6
+                     , crypton                  >= 0.31
                      , cryptostore              >= 0.2
```

Comment:
I would put `cryptostore >= 0.3.0.1`, when the `use_crypton` flag was introduced.